bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/I18nProviderImpl.java

```diff
@@ -90,13 +90,13 @@ protected void deactivate(ComponentContext componentContext) {
     }
 
     @Modified
-    protected synchronized void modified(Map<String, Object> config) {
-        final String language = (String) config.get(LANGUAGE);
-        final String script = (String) config.get(SCRIPT);
-        final String region = (String) config.get(REGION);
-        final String variant = (String) config.get(VARIANT);
-        final String location = (String) config.get(LOCATION);
-        final ZoneId timeZone = (ZoneId) config.get(TIMEZONE);
+    protected synchronized void modified(Map<String, String> config) {
```

Comment:
Is this allowed? The modified signature is imho defined by OSGi and it is totally valid to have values in the configuration that are not strings...